src/lib/statistics/chunk_statistics/histograms/histogram_utils.hpp

```diff
@@ -0,0 +1,104 @@
+#pragma once
```

Comment:
All over the histogram_utils I'd prefer `size_t` over `uint32_t`. If there is any reason to limit prefix lengths to 32bit, please use an alias and doc the reasons.